### ISSUE TITLE
Glue does not start on Windows 7

### ISSUE THREAD

**Issue by SebCanet** [gjthompson1/glue-public#5]
**Describe the bug**
When launching software, it says on animation with logo turning on.

**To Reproduce**
1. Launch software.

**Desktop (please complete the following information):**
 - OS: Windows 7 x64
 - App version v0.10.0
 
Thanks.

**Comment by HRo-Entropeak** [comment]
Not working on windows 10 as well.

```
check if server already running...
starting server...
Checking for update
Update for version 0.10.0 is not available (latest version: 0.10.0, downgrade is disallowed).
checkForUpdatesAndNotify called, downloadPromise is null
```

**Comment by gjthompson1** [comment]
Please refer to this: https://github.com/gjthompson1/glue-public/issues/9#issuecomment-884987402 for a general update which relates to this issue.
